bankapp/src/test/kotlin/com/clouway/bankapp/adapter/spark/BankEndToEndTest.kt

```diff
@@ -0,0 +1,290 @@
+package com.clouway.bankapp.adapter.spark
+
+import com.clouway.bankapp.core.Operation
+import com.clouway.bankapp.core.Transaction
+import com.google.appengine.repackaged.com.google.gson.Gson
+import com.google.appengine.repackaged.org.apache.http.client.methods.HttpGet
+import com.google.appengine.repackaged.org.apache.http.client.methods.HttpPost
+import com.google.appengine.repackaged.org.apache.http.client.methods.HttpUriRequest
+import com.google.appengine.repackaged.org.apache.http.entity.StringEntity
+import com.google.appengine.repackaged.org.apache.http.impl.client.BasicCookieStore
+import com.google.appengine.repackaged.org.apache.http.impl.client.HttpClientBuilder
+import com.google.appengine.repackaged.org.apache.http.util.EntityUtils
```

Comment:
try with https://github.com/google/google-http-java-client

Reply:
Rewrote the tests with google http client.